manifests/xinetd.pp

```diff
@@ -0,0 +1,18 @@
+# @summary Configure xinetd
+#
+# @param disable_xinetd_when_unused
+#   Disables xinetd if there are no services enabled under /etc/xinetd.d
+#
+# @example
+#   include profile_hardening::xinetd
+class profile_hardening::xinetd (
+  Boolean $disable_xinetd_when_unused,
+){
+  if ( $disable_xinetd_when_unused ) {
+    exec { 'disable_xinetd':
+      path    => ['/bin', '/usr/bin'],
+      command => "systemctl stop xinetd; systemctl disable xinetd",
+      unless  => "egrep -Riw '^\s*disable\s*=\s*no' /etc/xinetd.d",
```

Comment:
The double quotes should not be needed in these 2 lines.

For sure the first one. The 2nd one will require delimited the current single quotes to switch it to be in single quotes instead of double quotes.